app/registries/foreman/plugin/assets.rb

```diff
@@ -25,6 +25,15 @@ def automatic_assets(enabled)
         @automatic_assets = !!enabled
       end
 
+      def webpack_manifest_path
+        manifest_path = File.join(path, 'public', 'webpack', id.to_s, 'manifest.json')
+        File.file?(manifest_path) ? manifest_path : nil
+      end
+
+      def uses_webpack?
+        File.file?(File.join(path, 'webpack', 'index.js')) || webpack_manifest_path.present?
```

Comment:
This can result in a `nil.present?` which I think doesn't work.

Reply:
```
[10] pry(main)> false || nil.present?
=> false
```

Reply:
Interesting:
```
$ ruby --version
ruby 2.4.4p296 (2018-03-28 revision 63013) [x86_64-linux]
$ pry
[1] pry(main)> false || nil.present?
NoMethodError: undefined method `present?' for nil:NilClass
from (pry):1:in `__pry__'
```

Reply:
ah, that's because `.present?` comes from ActiveSupport :)

Reply:
Should we use the more standard `.nil?` instead?

Reply:
I think `.present?` is quite standard - there's 700 of them in our code :) It will also continue working if we decide to change the implementation of `webpack_manifest_path` to return an empty string in the future.